bindings/core/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "iota-sdk-bindings-core"
+version = "0.1.0"
+authors = [ "IOTA Stiftung" ]
+edition = "2021"
+homepage = "https://iota.org"
+description = "Bindings-core for the IOTA SDK"
+license = "Apache-2.0"
+
+[dependencies]
+iota-sdk = { path = "../../sdk", default-features = false, features = [ "client", "wallet", "tls", "message_interface" ] }
+
+backtrace = { version = "0.3.67", default-features = false, features = [ "std" ] }
+derivative = { version = "2.2.0", default-features = false}
+fern-logger = { version = "0.5.0", default-features = false }
+futures =  { version = "0.3.26", default-features = false }
+iota-crypto = { version = "0.15.3", default-features = false, features = [ "std", "chacha", "digest", "curl-p", "blake2b", "ed25519", "random", "slip10", "bip39", "bip39-en", "ternary_encoding" ] }
+log = { version = "0.4.17", default-features = false }
+packable = { version = "0.8.0", default-features = false, features = [ "serde", "primitive-types", "std" ] }
+prefix-hex = { version = "0.7.0", default-features = false, features = [ "std" ] }
+primitive-types = { version = "0.12.1", default-features = false, features = [ "serde" ] }
+serde = { version = "1.0.154", default-features = false, features = [ "derive" ] }
+serde_json = { version = "1.0.95", default-features = false }
+thiserror = { version = "1.0.40", default-features = false }
+zeroize = { version = "1.6.0", default-features = false, features = [ "zeroize_derive" ] }
+
+[dev-dependencies]
+tokio = { version = "1.27.0", default-features = false }
+
+[features]
+storage = [ "iota-sdk/storage" ]
+rocksdb = [ "iota-sdk/rocksdb" ]
+mqtt = [ "iota-sdk/mqtt" ]
+stronghold = [ "iota-sdk/stronghold" ]
+events = [ "iota-sdk/events"  ]
+ledger_nano = [ "iota-sdk/ledger_nano" ]
+participation = [ "iota-sdk/participation" ]
```

Comment:
I'm wondering if we need features at all for the bindings core. Eventually we enable them all in the different bindings.

Reply:
I mean, we have no feature gate mechanism in the different language so we need everything anyway.

Reply:
We have differences, participation is only in nodejs so far
participation and ledger nano won't work on wasm

Reply:
I see